YoCode/ProcessRunner.cs

```diff
@@ -5,26 +5,36 @@
 
 namespace YoCode
 {
-    class ProcessRunner
+    public class ProcessRunner
     {
+        private int timeout = 15000; //15 seconds
```

Comment:
Is 15 seconds long enough? I guess it should be. But another way to say "15 seconds" in C# would be `TimeSpan.FromSeconds(15)` and then you wouldn't need the comment.